frontend/packages/dev-console/console-extensions.json

```
@@ -1329,5 +1328,16 @@
       "path": ["/search-page/all-namespaces", "/search-page/ns/:ns"],
       "component": { "$codeRef": "common.SearchPage" }
     }
+  },
+  {
+    "type": "console.navigation/section",
+    "properties": {
+      "id": "ecosystem",
+      "perspective": "admin",
+      "section": "ecosystem",
+      "name": "%devconsole~EcoSystem%",
```

Casing issue

```suggestion
      "name": "%devconsole~Ecosystem%",
```
Plus, [we already have an `Ecosystem` `console.navigation/section` in OLM v1](https://github.com/openshift/console/blob/7273411ac2ddfebc7ba5e479795c67c63c65b536/frontend/packages/operator-lifecycle-manager-v1/console-extensions.json#L24-L35) that should be deduplicated.